atom/browser/native_window_mac.h

```diff
@@ -78,6 +78,8 @@ class NativeWindowMac : public NativeWindow {
     UpdateDraggableRegionViews(draggable_regions_);
   }
 
+  bool should_hide_native_toolbar_in_fullscreen_;
```

Comment:
The Google C++ style guide which Electron's code follows forbids public member datas, it has to be put in to private section and we can add a public getter for it. You can reference how we deal with `force_using_draggable_region_` in `native_window.h`.